Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.6-stretch
+
+RUN pip3 install git+https://github.com/Cobliteam/kafka-topic-dumper.git
+
+WORKDIR /opt/kafka-topic-dumper
```

Comment:
I don't think the package is actually installed in /opt. Does it make sense to set the work dir to there?